packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -617,10 +653,18 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ReinitializableBase
         }
     }
 
+    /// @notice Migrates the total ETH balance to the ETHLockbox.
+    function migrateLiquidity() public {
+        if (msg.sender != proxyAdminOwner()) revert OptimismPortal_Unauthorized();
+        _migrateLiquidity();
+    }
```

Comment:
Why do we need this? Presumably its handled in the `upgrade()` function. Are we trying to clean up 'force deposits' via selfdestruct?

Reply:
We need it because not all migrations will go via OPCM - most migrations will happen via superchain-ops tasks.